### PR TITLE
Awesome Books ES6

### PR DESCRIPTION
**Pull request by ohyajk**
I refactored the Awesome Books project code to use ES6  syntax and separated the code in modules to make it simple, short, and understandable to everyone.

### CODE REVIEW

File: modules/class.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+              `;
+      }
+    }
+  }
+
+    del(id) { //eslint-disable-line
+    getBooksag.splice(id, 1);
+    localStorage.setItem('books', JSON.stringify(getBooksag));
+  }
+}
```
Comment (ichala) [inline modules/class.js]: - [x] Good Job so far your project is awesome and everything works as it should, however, as part of the requirements we should stick with ES6 syntax, kindly update all of your BooksClass functions to an arrow functions example : 

```JS
  // Normal Function
  del(id) {
     ...
  }
  // Arrow Function ES6
   del = (id) => {
     ...
  }

```

File: modules/addBtn.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    if (title.value == "" || author.value == "") {//eslint-disable-line
+      alert('Please fill the required Fields'); //eslint-disable-line
+  } else {
+    const bookAd = new BooksClass();
+    bookAd.add();
+      alert('Book added successfully'); //eslint-disable-line
+    title.value = '';
+    author.value = '';
+  }
+};
```
Comment (ichala) [inline modules/addBtn.js]: - [x] for [javascript best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) kindly avoid using the javascript `alert()` method, t’s ugly, impossible to style, it stops code execution and displays differently on different browsers. you can put a custom error div and display your messages or create a modal for it , please fix it 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @ohyajk 
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Used Modules  ✔️ 
- No Linters Errors ✔️
- Application Works Perfectly ✔️
- Used Luxon ✔️


 ## Required Changes ♻️

- [ ] **_🛑  Check The Comments Below_**

## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/class.js (hunk 1 above)
Comment (ohyajk) [reply modules/class.js]: Thankyou for your feedback. I fixed it

File: modules/addBtn.js (hunk 2 above)
Comment (ohyajk) [reply modules/addBtn.js]: I have removed all alert prompts. Thankyou for your time.

**Review by ichala — APPROVED** [review]
Hi @ohyajk   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**